client/matching/peerResolver.go

```diff
@@ -20,28 +20,31 @@
 
 package matching
 
-import "github.com/uber/cadence/common/membership"
+import (
+	"github.com/uber/cadence/common/membership"
+	"github.com/uber/cadence/common/service"
+)
 
 // PeerResolver is used to resolve matching peers.
 // Those are deployed instances of Cadence matching services that participate in the cluster ring.
 // The resulting peer is simply an address of form ip:port where RPC calls can be routed to.
 type PeerResolver struct {
-	membership    membership.ServiceResolver
+	membership    membership.Resolver
```

Comment:
nit: this was renamed to `resolver` in `client/history/peerResolver.go`, maybe rename here as well for consistency?

Reply:
yep, can do